### PR TITLE
Refs Bug#391176 Fix some bug UI

### PR DESCRIPTION
**Pull request by tamtt-1031**
## Related Tickets

- [#391176](https://dev.sun-asterisk.com/issues/391176)

## What's this PR do ?

- [x] Fix some bug UI
- Update lại avatar current user, có hiển thị tên bên cạnh
- Cột email trong list user ko hiển thị dạng link
- User ko thể remove chính nó
- Một số link avatar ko hiển thị sau khi redirect
- Xóa icon thu nhỏ sidebar
- Xóa field search phía trên trang
- Thống nhất màu của các btn, kích thước
- Số thứ tự trong webhook list
## Library
*(List package, library third party add new include version)*

- N/A

## Performance

- [ ] Resolved n + 1 query
- [ ] Run explain query already
- [x] Time run rake task : 1000 ms

## Checklist

- [x] It was tested in local success?
- [x] Fill link PR into ticket and the opposite
- [x] Note purpose/reason, solution, scope of influence into ticket

## ENV note

```
```

## Notes
*(Other notes)*
Screenshot from 2019-12-05 15-48-38
Screenshot from 2019-12-05 15-48-49
Screenshot from 2019-12-05 15-49-01
Screenshot from 2019-12-05 15-49-16
Screenshot from 2019-12-05 16-14-44

### CODE REVIEW

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>failure</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>failure</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr></table>

File: app/helpers/helper.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+namespace App\Helpers;
+
+class Helper
```
Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR1.Methods.CamelCapsMethodName.NotCamelCaps</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 6</br>Method name "Helper::index_number" is not in camel caps format

Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 6</br>Variable "per_page" is not in valid camel caps format

File: app/helpers/helper.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+{
+    public static function index_number($page, $per_page, $index)
+    {
```
Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 9</br>Expected 1 space(s) after IF keyword; 0 found

Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 9</br>Newline required after opening brace

Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ScopeClosingBrace.ContentBefore</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 9</br>Closing brace must be on a line by itself

File: app/helpers/helper.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        $page = $page - 1;
+        if($page < 0) { $page = 0; }
```
Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 11</br>Variable "per_page" is not in valid camel caps format

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>failure</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>failure</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr></table>

File: app/helpers/helper.php (hunk 1 above)
Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR1.Methods.CamelCapsMethodName.NotCamelCaps</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 6</br>Method name "Helper::index_number" is not in camel caps format

Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 6</br>Variable "per_page" is not in valid camel caps format

File: app/helpers/helper.php (hunk 2 above)
Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 9</br>Expected 1 space(s) after IF keyword; 0 found

Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 9</br>Newline required after opening brace

Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ScopeClosingBrace.ContentBefore</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 9</br>Closing brace must be on a line by itself

File: app/helpers/helper.php (hunk 3 above)
Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 11</br>Variable "per_page" is not in valid camel caps format

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>failure</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>failure</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr></table>

File: app/helpers/helper.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?php
+namespace App\Helpers;
+
+class Helper
+{
+    public static function indexNumber($page, $perPage, $index)
+    {
```
Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 9</br>Expected 1 space(s) after IF keyword; 0 found

Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 9</br>Newline required after opening brace

Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ScopeClosingBrace.ContentBefore</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 9</br>Closing brace must be on a line by itself

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>failure</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr></table>

File: app/helpers/helper.php [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+        $page = $page - 1;
```
Comment (framgiaciv3) [inline app/helpers/helper.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/helpers/helper.php</br>**Line**: 10</br>Expected 1 space(s) after IF keyword; 0 found

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr></table>

**Review by nguyenthinhhn — APPROVED** [review]

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr></table>

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr></table>

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr></table>

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: resources/views/admins/webhooks/index.blade.php [hunk 6]
```diff
@@ -42,6 +43,7 @@
             <tbody>
                 @foreach ($webhooks as $webhook)
                     <tr class="item-{{ $webhook->id }}">
+                        <td>{{ Helper::indexNumber(app('request')->input('page'), 15, $loop->iteration) }}</td>
```
Comment (ghost) [inline resources/views/admins/webhooks/index.blade.php]: thay số 15 bằng config('paginate.perPage') đi a

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr></table>

**Comment by phucqd-1305** [comment]
OK

**Pull request merged**